src/lib/components/Swap/Swap.fixture.tsx

```diff
@@ -24,7 +25,34 @@ function Fixture() {
     }
   }, [color, setColor])
 
-  return <Swap defaults={{ tokenList: tokens }} />
+  const addressOptions = ['', 'NATIVE', DAI.address, USDC.address]
```

Comment:
I don't think these defaults need to be a part of cosmos. Can they be removed, and instead included in the demo site that we produce for integrators? wdyt?

Reply:
They're useless in cosmos at runtime, actually, but included for completeness. Is there a way to do object mapping for select options in cosmos? I also wanted something cleaner than addresses, but didn't see anything when I skimmed the docs.